src/test/java/org/openhab/binding/zwave/test/internal/protocol/commandclass/ZWaveClimateControlScheduleCommandClassTest.java

```diff
@@ -1,6 +1,6 @@
 /**
  * Copyright (c) 2014-2016 by the respective copyright holders.
- *
+ * <p>
```

Comment:
Why has this been added? Did you run an mvn license (or whatever the command is)? I don't think it's consistent is it?